src/Adapters/Facebook/OEmbed.php

```diff
@@ -21,6 +21,16 @@ protected function detectEndpoint(): ?UriInterface
         }
 
         $uri = $this->extractor->getUri();
+        if (strpos($uri->getPath(), 'login') !== false) {
+            $query = $uri->getQuery();
+            $params = explode('&', $query);
```

Comment:
Please, instead parsing the query manually, use [`parse_str` function](https://www.php.net/manual/en/function.parse-str)

Reply:
Updated, see below. 
Thanks for pointing out this function (I use a lot of php but usually a bit higher level, so didn't know it existed).